### ISSUE TITLE
Having error with Soundcloud

### ISSUE THREAD

**Issue by miketamis** [mopidy/mopidy-soundcloud#59]
``` python
ERROR    SoundCloudBackend backend caused an exception.
Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/mopidy/core/library.py", line 19, in _backend_error_handling
    yield
  File "/usr/lib/python2.7/dist-packages/mopidy/core/library.py", line 112, in _browse
    result = backend.library.browse(uri).get()
  File "/usr/lib/python2.7/dist-packages/pykka/threading.py", line 52, in get
    compat.reraise(*self._data['exc_info'])
  File "/usr/lib/python2.7/dist-packages/pykka/compat.py", line 12, in reraise
    exec('raise tp, value, tb')
  File "/usr/lib/python2.7/dist-packages/pykka/actor.py", line 201, in _actor_loop
    response = self._handle_receive(message)
  File "/usr/lib/python2.7/dist-packages/pykka/actor.py", line 295, in _handle_receive
    return callee(*message['args'], **message['kwargs'])
  File "/usr/lib/python2.7/dist-packages/mopidy_soundcloud/library.py", line 166, in browse
    return self.list_liked()
  File "/usr/lib/python2.7/dist-packages/mopidy_soundcloud/library.py", line 78, in list_liked
    logger.debug('Adding liked track %s to vfs' % data.name)
AttributeError: 'list' object has no attribute 'name'
```

**Comment by blueyed** [comment]
This happens when accessing the "Liked" items.

I get this INFO in the log before:

> INFO     'X' can't be streamed from SoundCloud

**Comment by blueyed** [comment]
Removing the like on the track fixes the issue.

The track is https://soundcloud.com/andhim/andhim-live-at-fusion.

**Comment by magikcypress** [comment]
I have the same bug with this track:

https://soundcloud.com/andhim/wine-and-chocolate-andhim-rmx

**Comment by jodal** [comment]
Fixed by #69.
